package.json

```diff
@@ -52,52 +50,52 @@
     "sinon": "^12.0.0",
     "temp": "^0.9.1",
     "tslint": "^5.12.0",
-    "typedoc": "^0.22.11",
+    "typedoc": "^0.26.11",
     "typedoc-plugin-external-module-map": "1.3.2",
     "typescript": "~5.4.5",
     "yargs": "^15.3.1"
   },
   "scripts": {
-    "all": "yarn -s install && yarn -s lint && yarn -s build",
-    "browser": "yarn -s --cwd examples/browser",
-    "browser-only": "yarn -s --cwd examples/browser-only",
-    "build": "yarn -s compile && yarn -s build:examples",
-    "build:examples": "yarn browser build && yarn electron build && yarn browser-only build",
-    "clean": "yarn -s rebuild:clean && yarn -s lint:clean && node scripts/run-reverse-topo.js yarn -s clean",
-    "compile": "echo Compiling TypeScript sources... && yarn -s compile:clean && yarn -s compile:tsc",
-    "compile:clean": "ts-clean dev-packages/* packages/*",
-    "compile:references": "node scripts/compile-references.js",
-    "compile:tsc": "tsc -b",
+    "all": "npm -s install && npm run -s lint && npm run -s build",
+    "build": "npm run compile && npm run build:applications",
```

Comment:
The trouble is that you can't really tell which one is the "most common" case. I, for example, usually do my testing in the electron variant. It's also consistent with the current behavior (+ building the new application "browser-only").

Reply:
That's fine with me. Personally I don't like `build` without `build:all` to build everything as building all applications takes a long time.
However this might be fixed anyway with https://github.com/eclipse-theia/theia/pull/14414